docs/src/content/docs/sdk.mdx

```diff
@@ -84,6 +84,42 @@ sendText(text: string): void
 ```
 Sends a message to the model. Model replies via text transcript. Requires inputting the text message (string).
 
+#### registerToolImplementation()
+```ts
+registerToolImplementation(name: string, implementation: ClientToolImplementation): void
+```
+Registers a client tool implementation with the given name. If the call is started with a client-implemented tool, this implementation will be invoked when the model calls the tool.
+
+| parameter        | description                                 |
+|----------------- | ------------------------------------------- |
+| name | String. The name of the tool. Must match what is defined in `selectedTools` during [call creation](../api/calls/#create-call). |
```

Comment:
In particular it should match the nameOverride if set and the modelToolName otherwise.

Reply:
added